app/views/groups/_group-movement.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<div class="expenses">
+  
+  <div class="expenses-list">
+
+  <%# <% @group.movements.each do |expense| %>
+    <%# <%= render expense %>
+  <%# <% end %>
+
+  </div>
+  <button class="">
+  <%# <%= link_to "New Expense", new_category_expense_path(@category), class: '' %>
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
@BenMukebo You are absolutely right, thank you very much for your comments. I have already applied your comments and applied your advice.

Reply:
This change has been done well